lib/fs.ts

```diff
@@ -21,80 +21,72 @@
  * @license MIT
  */
 
-'use strict';
+import * as fs from 'fs';
+import { dirname as pathDirname, resolve as pathResolve } from 'path';
```

Comment:
idk how to feel about this, but `pathModule` was also pretty ugly.

Reply:
Yeah, I figured this was nicer than `pathModule`? Happy to revert though.